src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -371,6 +372,43 @@ else if (!vc.getAlternateAllele(0).equals(Allele.NON_REF_ALLELE)){      //there
         }
     }
 
+    /*
+     * Test that in VCF mode we're consistent with past GATK4 results
```

Comment:
Comment does not match the test. Should be something like "Test the `--floor-blocks` argument."